xbdd-web/src/modules/summary/productList/ProductListItem.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { List, Checkbox, ListItem, ListItemText, ListItemIcon, Collapse } from "@material-ui/core";
+import Favorite from "@material-ui/icons/Favorite";
+import FavoriteBorder from "@material-ui/icons/FavoriteBorder";
+import ExpandLess from "@material-ui/icons/ExpandLess";
+import ExpandMore from "@material-ui/icons/ExpandMore";
+import { withStyles } from "@material-ui/core/styles";
+import ProductListStyles from "./styles/ProductListStyles";
+import Product from "../../../models/Product";
+
+const ProductListItem = props => {
+  function clickEventWrapper(event) {
```

Comment:
Can we move this to a function outside of the ProductListItem function. having functions in functions like this generally makes code a little harder to read. This case isn't so bad but I think it would be better to follow the pattern, also change it to a arrow function for consistency.